ts/features/walletV3/transaction/screens/WalletTransactionOperationDetails.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from "react";
+import { ScrollView } from "react-native";
+import {
+  Divider,
+  H1,
+  H6,
+  IOStyles,
+  ListItemInfo
+} from "@pagopa/io-app-design-system";
+import { RouteProp, useRoute } from "@react-navigation/native";
+
+import { WalletTransactionParamsList } from "../navigation/navigator";
+import TopScreenComponent from "../../../../components/screens/TopScreenComponent";
+import { Dettaglio } from "../../../../../definitions/pagopa/Dettaglio";
+import { formatNumberCentsToAmount } from "../../../../utils/stringBuilder";
+import { cleanTransactionDescription } from "../../../../utils/payment";
+import I18n from "../../../../i18n";
+
+export type WalletTransactionOperationDetailsScreenParams = {
+  operationName: string;
+  operationDetails: Dettaglio;
+};
+
+export type WalletTransactionOperationDetailsScreenProps = RouteProp<
+  WalletTransactionParamsList,
+  "WALLET_TRANSACTION_OPERATION_DETAILS"
+>;
+
+const WalletTransactionOperationDetailsScreen = () => {
+  const route = useRoute<WalletTransactionOperationDetailsScreenProps>();
+  const { operationDetails, operationName } = route.params;
+
+  const getDebitoreText = () => {
+    const debitoreNameLabel = operationDetails.nomePagatore ? (
+      <H6>{operationDetails.nomePagatore}</H6>
+    ) : (
+      <></>
+    );
+    const debitoreCodiceLabel = operationDetails.codicePagatore ? (
+      <H6>({operationDetails.codicePagatore})</H6>
+    ) : (
+      <></>
+    );
+    return (
+      <>
+        {debitoreNameLabel}
+        {debitoreCodiceLabel}
+      </>
+    );
+  };
```

Comment:
I think this function should be completely in english; what do you think?

Reply:
Yep I agree! Addressed into 31b14c92e3ed6db043a8f05f9d4cdd3d93246a84